examples/DocQA/docker/llama_stack_start.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+sleep 45
+echo "-----starting to llama-stack docker now---------"
+pip install gradio
+
+if [ "$USE_GPU_FOR_DOC_INGESTION" = true ]; then
```

Comment:
looks like if that variable is false, we just don't ingest at all?

Reply:
Yes, ingest pipeline is just for image: basically it takes a PDF, find out if there is any image and use 11B to generate image description, then it will save the original text with image description into `output` folder so that everything is now text data, ready to be used by 3B RAG agent. Running 11B on CPU is sooo slow and we have not enable this in the current stage. We believe this can be a P1 feature to have.